IOStream/FileInputStream.m

```diff
@@ -36,8 +37,24 @@ - (void)dealloc {
     [self close];
 }
 
+- (void)enableCopyBuffer:(BOOL)enable {
```

Comment:
I think a getpos method makes total sense in FileInputStream.

I'm open to other ways to implement this as well, seek was just the first thing to pop into my head. For example, you could write a CrcInputStream that wraps the FileInputStream and calculates the crc as the file is read. That's a lot more work though, and I think the seek if fine if you decide to implement it that way.